Makefile

```diff
@@ -856,7 +856,7 @@ $(UTILDIR)/megaflash-a100t.prg:       $(UTILDIR)/megaflash.c $(UTILDIR)/qspicomm
 	$(info =============================================================)
 	$(info ~~~~~~~~~~~~~~~~> Making: $@)
 	$(CL65) -I $(SRCDIR)/mega65-libc/cc65/include -DA100T -O -o $(UTILDIR)/megaflash-a100t.prg \
-		--add-source -Ln $*.label --listing $*.list \
```

Comment:
I'm not sure who made it .label or what their intention was behind that. As far as I know, I introduced support for m65dbg first and it was .lbl there, so feel like someone else has supported another debugger that prefers .label.

As for which should take precedence, can't say. Feel like somebody pulled the rug from under my feet and now I'm doing it back to them. Ah well, maybe in future, I can get m65dbg to support either .lbl or .label.

On the other hand, whoever needed .label instead of .lbl could do likewise in their debugger.